flask_security/core.py

```diff
@@ -425,6 +425,9 @@ def init_app(self, app, datastore=None, register_blueprint=True,
         """
         datastore = datastore or self.datastore
 
+        email_sender = app.config.get('MAIL_DEFAULT_SENDER')
+        _default_config['EMAIL_SENDER'] = email_sender or 'no-reply@localhost'
```

Comment:
Yes, you are right. Modifying ```_default_config``` may initialize other apps incorrectly. I have updated the PR.